src/main/java/seedu/duke/controllers/ModulePlannerController.java

```diff
@@ -59,18 +64,34 @@ public void start() {
         initialiseUser();
         displayReady();
         handleUserInputTillExitCommand();
+        saveStudentData();
         displayGoodbye();
     }
 
     public void initialiseUser() {
         Scanner in = new Scanner(System.in);
         String userInput;
         do {
-            System.out.println("Please enter your name: ");
+            System.out.println("Please enter your name (used to retrieve your save file): ");
             userInput = in.nextLine().trim();
         } while (!parser.checkNameInput(userInput, commandManager.getListOfCommandNames()));
         student.setName(userInput);
 
+        // Create storage file based on userName
+        storage = new Storage(userInput);
+        try {
```

Comment:
initialiseUser function seems to pile up with a lot of logic(do,while and try catches) which may decrease readability. Maybe we could attract out the different stages in initialiseUser to improve readability?